bookkeeper-server/pom.xml

```diff
@@ -37,6 +37,18 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.apache.bookkeeper</groupId>
+      <artifactId>bookkeeper-common-allocator</artifactId>
+      <version>${project.parent.version}</version>
+      <exclusions>
+        <!-- exclude "netty-buffer" since "netty-all" already introduces it -->
```

Comment:
Good point, will fix that. In general we should also refactor netty dependency to not use netty-all, but that's a completely different story

Reply:
making `provided` will be another problem. because everyone who uses `bookkeeper-common-allocator` will have to manually add `netty-buffer`. I think the right approach is refactoring netty dependency usage, although it can be a separated PR.